app/controllers/beers_controller.rb

```diff
@@ -0,0 +1,55 @@
+class BeersController < ApplicationController
+  def index
+    @beers = Beer.page(params[:page]).order(:name)
+  end
+
+  def show
+    @beer = Beer.find(params[:id])
+  end
+
+  def new
+    @beer = Beer.new
+  end
+
+  def create
+    @beer = Beer.new(beer_params)
+    if @beer.save
+      flash[:notice] = "Beer was saved"
+      redirect_to 'beer#show'
+    else
+      flash[:notice] = "Beer not added"
+      render :new
+    end
+  end
+
+  def edit
+    @beer = Beer.find(params[:id])
```

Comment:
This will allow any user to edit any beer. To scope to beers that I own:

```ruby
def edit
  @beer = Beer.find_by!(user: current_user, id: params[:id])
  # alternatively
  @beer = current_user.beers.find(params[:id])
end
```